shared/parser.go

```diff
@@ -351,7 +351,7 @@ func Parse(argList []string) ParserResult {
 		"-fopenmp":       {0, pr.compileLinkUnaryCallback},
 
 		"-Wl,-dead_strip": {0, pr.warningLinkUnaryCallback},
-		"-dead_strip":     {0, pr.warningLinkUnaryCallback}, //iam: tor does this. We lose the bitcode :-(
+		"-dead_strip":     {0, pr.warningLinkUnaryCallback},  //iam: tor does this. We lose the bitcode :-(
```

Comment:
I don't think the ` ` addition is intentional?

Reply:
FIx in [7a7facb](https://github.com/capablevms/gllvm/pull/3/commits/7a7facb04ac073ce43dfbd95f6d72eeabc4b66cb).